docker/README.md

```diff
@@ -1,40 +1,33 @@
-# SyTest Docker
+# SyTest Docker Images
 
-Herein lies a Dockerfile for building a functional SyTest test environment.
-SyTest and synapse are cloned from the HEAD of their develop branches. You can
-run the tests as follows:
+These Dockerfiles create containers for running SyTest in various configurations.
 
-```
-cd /path/to/sytest/docker
-docker build -t sytest .
-docker run --rm -it sytest bash
-```
+Included currently is:
 
-And then at the shell prompt:
-
-```
-./run-tests.pl
-```
+- matrixdotorg/sytest, a base container with SyTest dependencies installed
```

Comment:
one thing that I didn't quite grok at first was that the docker images don't in fact include Sytest itself, which will be pulled from git. I realise it's alluded to below, but it might be worth making explicit?